modules/ROOT/pages/apikit-4-set-header-task.adoc

```diff
@@ -3,39 +3,81 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you add header information to the response that complies with the WSDL. Header information that complies with the WSDL document is called APIUsageInformation.
+Using APIkit you can set headers to your application's response based on the WSDL declarations.
 
-. In Studio, open the tshirt2.wsdl in `src/main/resources/api` and scroll to APIUsageInformation element, which is the element expected by the contract. Copy the name of the element to the clipboard.
-. On the canvas, select Transform message in the `OrderTshirt:/soapkit-config` flow, and in the properties editor, click Add New Target:
-+
-image::apikit-for-soap-3005f.png[height=440,width=700]
-+
-. Select Variable from the drop-down.
-+
-. In Variable Name, paste the contents of the clipboard, and add *soap.* as a prefix. or type *soap.APIUsageInformation*.
-+
-The complete variable name looks like this:
-+
-soap.APIUsageInformation
-+
-. Click OK.
-. In Output, double-click apiCallsRemaining: _Integer_.
+To follow this example, you must have downloaded the `tshirt2.wsdl` API definition, and created a Mule application based on it. +
```

Comment:
I suggest: "To follow this example, you must first download the `tshirt2.wsdl` API definition and create a Mule application based on that definition."